ecpy_pulses/api.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by EcpyPulses Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Ecpy Pulses Plugin public interface.
+
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+
+from .sequences.base_sequences import BaseSequence, RootSequence
+from .pulse import Pulse
+from .item import Item
```

Comment:
Contributions to the manifest should be listed here too (so Shape, BaseShape, Context, BaseContext, Sequence, ....)